src/cli/node/with_node.rs

```diff
@@ -629,6 +630,12 @@ pub enum Command {
         #[structopt(short = "e", long = "epoch")]
         epoch: Epoch,
     },
+    #[structopt(name = "tapiInfo", about = "Get Information related to TAPI signaling")]
```

Comment:
I prefer the name `signalingInfo` because TAPI probably doesn't mean anything to node operators.

```suggestion
    #[structopt(name = "signalingInfo", about = "Get Information related to TAPI signaling")]
```